app/lib/import_export_constants.rb

```diff
@@ -387,6 +405,19 @@ module ImportExportConstants # rubocop:todo Metrics/ModuleLength
     updated_at: 'Close Contact Updated Date'
   }.freeze
 
+  COMMON_EXPOSURE_COHORT_FIELD_NAMES = {
+    patient_id: 'Sara Alert ID',
+    user_defined_id_statelocal: 'State/Local ID',
+    user_defined_id_cdc: 'CDC ID',
+    user_defined_id_nndss: 'NNDSS ID',
+    id: 'Common Exposure Cohort ID',
+    cohort_type: 'Cohort Type',
+    cohort_name: 'Cohort Name/Description',
+    cohort_location: 'Cohort Location',
+    created_at: 'Close Contact Created Date',
+    updated_at: 'Close Contact Updated Date'
```

Comment:
These should be "Common Exposure Cohort Created Date" and "... Updated Date"

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/11f9765d3d0d6b687f0a3e8f6d62ab6192eadf4b#diff-850172f7658c1cabc6010b0d7e23c668251b024c7aad38651260e458ecee9478R417